pkg/cmd/codespace/view.go

```diff
@@ -126,8 +111,19 @@ func (a *App) ViewCodespace(ctx context.Context, opts *viewOptions) error {
 
 func formatGitStatus(codespace codespace) string {
 	branchWithGitStatus := codespace.branchWithGitStatus()
-	// u2193 and u2191 are the unicode arrows for down and up
-	return fmt.Sprintf("%s - %d\u2193 %d\u2191", branchWithGitStatus, codespace.GitStatus.Behind, codespace.GitStatus.Ahead)
+
+	// Format the commits ahead/behind with proper pluralization
+	commitsAhead := "commit"
```

Comment:
We got a small little helper for pluralization that we could use for the ahead and behind commit(s).
```suggestion
	commitsAhead := text.Pluralize(codespace.GitStatus.Ahead, "commit")
```

Reply:
Didn't know about that, thanks! Updated it.